code/game/objects/structures/lavaland/legionturret.dm

```diff
@@ -0,0 +1,90 @@
+///A basic turret that shoots at nearby mobs. Intended to be used for the legion megafauna.
+/obj/structure/legionturret
```

Comment:
I really don't like this. Turrets are perf heavy and copy pasting version that's intended to be spammable is horrible idea.

Reply:
Alright, I removed them from processing. They now only shoot once and then get deleted.